site/content/docs/traces-open-telemetry/exporting-traces/grafana-tempo.md

```diff
@@ -13,14 +13,34 @@ beta: true
 aliases:
   - "/docs/open-telemetry/exporting-traces/grafana-tempo/"
 ---
+You can connect the Traces export to Grafana using the OpenTelemetry Integration and Grafana Tempo.
 
-1. Make sure you have Grafana Tempo installed, running and accessible. If you are running Grafana Cloud, the Tempo
-   datasource is already pre-configured. Find it at **Connections** > **Datasources** > **Tempo**.
-2. After installing, grab the Tempo endpoint URL. It should look like `https://tempo-eu-west-0.grafana.net/tempo`.
-3. Under the section **Authentication**, grab the user and password for the Tempo endpoint.
-4. As Tempo uses Basic Authentication, you need to provide the user and password in a `Authorization: Basic user:password` HTTP header in
-   Checkly integration settings, where the `user:password` section is base64 encoded. You can use an online tool like [base64encode.net](https://www.base64encode.net/)
-   to encode your user and password.
+1. Make sure that you have Tempo running on your Grafana Cloud Instance at `yourOrganization.grafana.net`. Find it at **Connections > Search for Tempo > Check if Tempo Data source is marked as installed**.
+   
+1. In Grafana Cloud (`grafana.com/orgs/yourOrganization`), head over to **Connections** > **Add new connection** > **OpenTelemetry (OTLP)**.
+  ![Add an OTLP connection in Grafana Cloud](/docs/images/otel/export-traces/grafana-cloud-opentelemetry.png)
 
-Find all the details in the [Grafana Tempo OpenTelemetry documentation](https://grafana.com/docs/tempo/latest/).
+1. Copy the endpoint URL and append `v1/traces` to it.
+  ![Grafana OTLP endpoint](/docs/images/otel/export-traces/grafana-otlp-endpoint-config.png)
+  It should be similar to:`https://otlp-gateway-prod-eu-west-2.grafana.net/otlp/v1/traces`.
+   
+1. Generate an OTLP Token, and copy both the Instance ID and the OTLP Token as well.
+  ![Generate an OTLP token](/docs/images/otel/export-traces/create-otlp-token-grafana.png)
 
+1. Head back to the [Traces Settings page](https://app.checklyhq.com/settings/account/traces) on Checkly.
+
+  * Enable exporting traces and add the endpoint URL from step 3.
+  * and in the **Headers** section, specify the HTTP Header: on the left column `Authorization` and `Basic instance:token`, where `instance:token` is base64 encoded.
```

Comment:
One last thing I tripped over: What do you mean by "on the left column"?

If I understand correctly `Authorization` should be added in the header field and `Basic instance:token` from step 4 in the value field, right?

Reply:
yes!